src/components/organisms/OcResource/OcResource.vue

```diff
@@ -8,7 +8,7 @@
       width="40"
       height="40"
     />
-    <oc-icon v-else key="resource-icon" :name="resource.icon" size="large" />
+    <oc-resource-icon :resource="resource"></oc-resource-icon>
```

Comment:
```suggestion
    <oc-resource-icon v-else :resource="resource" />
```

Reply:
a) `v-else` is needed, otherwise you'll show previews and icons side by side whenever a preview exists. If a preview exists the icon needs to stay hidden.
b) can be written as a self closing tag